metricbeat/module/docker/event/event.go

```diff
@@ -122,15 +122,11 @@ func (m *MetricSet) reportEvent(reporter mb.PushReporterV2, event events.Message
 
 	reporter.Event(mb.Event{
 		Timestamp: time,
-		RootFields: common.MapStr{
-			"event": common.MapStr{
-				"kind":    event.Type,
-				"action":  event.Action,
-				"created": time,
-				"id":      event.ID,
-			},
-		},
 		MetricSetFields: common.MapStr{
+			"id":     event.ID,
+			"type":   event.Type,
+			"action": event.Action,
+			"time":   event.Time,
```

Comment:
`time` is probably the one I would move to ECS. I wonder if this should even be `@timestamp`?

Reply:
yes, I will just remove it from here, as it's already reported under @timestamp here: https://github.com/elastic/beats/blob/48d2295fdaea431e59b2d6b42a59044de02d5722/metricbeat/module/docker/event/event.go#L124?